src/style.css

```
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bigSearchBar {
+  margin-top: 20px;
+}
+
+.smallSearchBar {
+  margin-top: 20px;
+}
+
+.search__input {
```

Use consistent cases for your classnames, better to choose snake_case mixed with kebab-case
```suggestion
.small-searchbar {
  margin-top: 20px;
}

.search__input {
```